addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -114,7 +114,7 @@
                     self.currentVideoTime += 1;
                     if (self.totalVideoTime && self.currentVideoTime > self.totalVideoTime - 30){
                         clearInterval(self.tid);
-                        if (!self.slide.hasQuestion && !self.slide.completed){
+                        if (self.slide && !self.slide.hasQuestion && !self.slide.completed){
```

Comment:
Hello @abla001 ,
Thanks for the quick reply, but this doesn't look right... How does this prevent triggering the event when the user is not a member?

Reply:
@flch-odoo sorry my bad, it was a typo I forgot to add `isMember` and my tests succeeded because I left my first fix there :sweat_smile:

Reply:
@flch-odoo can we r+ if everything is ok ?

Reply:
@flch-odoo